src/dev/jest/setup/polyfills.js

```diff
@@ -30,9 +30,8 @@
 
 // bluebird < v3.3.5 does not work with MutationObserver polyfill
 // when MutationObserver exists, bluebird avoids using node's builtin async schedulers
-const bluebird = require('bluebird');
-bluebird.Promise.setScheduler(function (fn) {
-  global.setImmediate.call(global, fn);
+queueMicrotask(() => {
```

Comment:
I haven't quite figured it out, so I'm not sure what the right solution is, can you tell me what the right solution should be?

Reply:
I think you could just removing Bluebird here. The scheduler will no longer be set to use setImmediate, and will use the default native Promise scheduling instead. Unless the code depends on the specific scheduling behavior of Bluebird, this should not be a problem.

```
const MutationObserver = require('mutation-observer');
Object.defineProperty(window, 'MutationObserver', { value: MutationObserver });

require('whatwg-fetch');
```

Reply:
Anan is right; being a Jest polyfill, we will know if we have a problem instantly due to test failures.